spec/sidekiq/hca/ezr_submission_job_spec.rb

```diff
@@ -11,6 +11,7 @@
     HealthCareApplication::LOCKBOX.encrypt(form.to_json)
   end
   let(:ezr_service) { double }
+  let(:tags) { described_class::DD_ZSF_TAGS }
```

Comment:
I like being able to use the value from the described_class here. Makes any text changes easy. 👍